tools/decompress-assemblies/decompress-assemblies.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">  
+  <PropertyGroup>
+    <Company>Microsoft Corporation</Company>
+    <Copyright>2021 Microsoft Corporation</Copyright>
+    <Version>0.0.1</Version>
+    <TargetFramework>net472</TargetFramework>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <RootNamespace>Xamarin.Android.Tools.DecompressAssemblies</RootNamespace>
+    <AssemblyName>decompress-assemblies</AssemblyName>
+    <OutputPath>../../bin/$(Configuration)/bin</OutputPath>
+    <OutputType>Exe</OutputType>
+    <LibZipSharpBundleAllNativeLibraries>true</LibZipSharpBundleAllNativeLibraries>
+    <LangVersion>$(LangVersion)</LangVersion>
```

Comment:
I don't think you need to set `$(LangVersion)` with `$(LangVersion)`.

This imports `Configuration.props`, so I think it would get the value you want;

https://github.com/xamarin/xamarin-android/blob/1877dd5ebee90452c6ab2cc33a58498a83ea7c1f/Configuration.props#L38